internal/provider/user_resource.go

```diff
@@ -3,6 +3,7 @@ package provider
 import (
```

Comment:
Changes in this file are to handle a change in the api library caused by a change on the Unleash side. Not relevant to this PR but needed to make the tests pass